packages/core-react/.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    parser: "@typescript-eslint/parser", // Specifies the ESLint parser
```

Comment:
We should create another repo to store our configuration, so we can use it all our projects. based and like : https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb

Reply:
I think your absolutely right. But as a 1st step I believe that this addition is already an improvement. I propose creating a ticket for what you suggest. What do you think of that?